Jenkinsfile

```diff
@@ -141,26 +36,24 @@ node {
             string(credentialsId: 'albArn', variable: 'albArn')
           ]) {
           def extraCommands = "--values ./helm/ffc-demo-web/jenkins-aws.yaml --set name=ffc-demo-$containerTag,ingress.server=$ingressServer,ingress.endpoint=ffc-demo-$containerTag,ingress.alb.tags=\"$albTags\",ingress.alb.arn=\"$albArn\",ingress.alb.securityGroups=\"$albSecurityGroups\""
-          deployPR(kubeCredsId, registry, imageName, containerTag, extraCommands)
+          defraUtils.deployChart(kubeCredsId, registry, imageName, containerTag, extraCommands)
           echo "Build available for review at https://ffc-demo-$containerTag.$ingressServer"
         }
       }
     }
     if (pr == '') {
       stage('Publish chart') {
-        publishChart(registry, imageName, containerTag)
+        defraUtils.publishChart(registry, imageName, containerTag)
       }
     }
     if (mergedPrNo != '') {
       stage('Remove merged PR') {
-        sh "echo removing deployment for PR $mergedPrNo"
-        undeployPR(kubeCredsId, imageName, mergedPrNo)
+        defraUtils.undeployChart(kubeCredsId, imageName, mergedPrNo)
       }
     }
-    updateGithubCommitStatus('Build successful', 'SUCCESS', repoUrl, commitSha)
+    defraUtils.updateGithubCommitStatus('Build successful', 'SUCCESS')
```

Comment:
I've been considering if we should have 3 methods for this, instead of passing string statuses in, I'm not keen on magic values.

Reply:
Good point, there are some other statuses too, like UNKNOWN that we wouldn't use. 
The fewer parameters the better.

Might be worthwhile doing it now before all the other branches are merged. i.e. leave the `updateGithubCommitStatusupdate` as it is but add `setGithubStatusSuccess`, `setGithubStatusPending` and `setGithubStatusFailure`

Reply:
Good idea. I've created a new branch to do that, and I'm testing against the user-service.

Reply:
updated code to use new github methods